docs/README.md

```diff
@@ -307,25 +306,41 @@ This service account needs to have permission to access the Firestore data.
 10. Save and deploy the code.
 
 
-## Advanced Use-cases
+## Value Calculation
 
-### Combine Return Rate with Profit
+There are different methods for calculating the value built into the tag.
 
-If some products are returned more than others, you could calculate the return
-rate percentage at a product level.
+- `Value`: This is the default method. The calculation is simply:
+  ```
+  conversion_value = profit * quantity
+  ```
+- `Return Rate`: If some products are returned more than others, you could
+  calculate the return rate percentage at a product level.
 
-Then in Firestore you can add the document with both the profit and return rate:
+  Then in Firestore you can add the document with both the profit and return
+  rate:
 
-![Firestore screenshot with return rate](./img/firestore-with-return-rate.png)
+  ![Firestore screenshot with return rate](./img/firestore-with-return-rate.png)
 
-And use the [firestore-value-return-rate-template.tpl](
-./../src/gtm/firestore-value-return-rate-template.tpl) to factor in both these
-values when calculating the conversion value. The template performs the
-following calculation for each product:
+  ```
+  conversion_value = (1 - return_rate) * profit * quantity
+  ```
+  If you select this field, you can optionally override the name of the return
```

Comment:
Just a nit: 

Would this read better as "If you select this **option**, you can optionally override the name of the return rate field in Firestore." - or maybe I'm misunderstanding something

Reply:
Fair point. I've updated this